src/Conversion/ONNXToKrnl/Math/Elementwise.cpp

```diff
@@ -1025,7 +1021,7 @@ static LogicalResult getVariadicSimdCodeFullyFlattened(
   unsigned numArgs = op->getNumOperands();
 
   if (DEBUG)
-    fprintf(stderr, "SIMD code for variadic op %s\n", getOpName(op));
+    llvm::errs() << "SIMD code for variadic op " << op->getName() << "\n";
```

Comment:
Is there any particular reason to use `errs()` instead of `outs()`?

Reply:
yes, it is just a way for me to enable/disable simd for the ops that show up in a model.